Mathlib/Order/LiminfLimsup.lean

```diff
@@ -1517,3 +1517,98 @@ theorem liminf_finset_inf [ConditionallyCompleteLinearOrder β] [OrderTop β] {f
   limsup_finset_sup (β := βᵒᵈ) h₁ h₂
 
 end MinMax
+
+section frequently_bounded
+
+variable {R S : Type*} {F : Filter R} [LinearOrder R] [LinearOrder S]
+
+/-- For nontrivial filters in linear orders, coboundedness for `≤` implies frequently boundedness
```

Comment:
I think "frequent boundedness" might make sense, but adverbs can't really modify nouns like this. Perhaps a complete rewrite of the sentence would be an alternative. Applies here and throughout.